src/scss/components/_buttons.scss

```diff
@@ -27,6 +27,49 @@
   }
 }
 
+.usa-button--unstyled {
+  position: relative;
+  text-underline-offset: 3px;
+
+  // In USWDS, unstyled button styles are applied last to take precedence over base button styles.
+  // Since we override base button styles, we must re-apply the unstyled button styles.
+  &.usa-button {
+    @include button-unstyled;
+  }
+
+  // Hover colors are already applied by USWDS via `typeset-link`, but this does not include the
+  // button modifier classes.
+  &:hover,
+  &.usa-button--hover {
+    color: color($theme-link-hover-color);
+  }
+
+  &:active,
+  &.usa-button--active {
+    color: color($theme-link-active-color);
+  }
```

Comment:
@anniehirshman-gsa I noticed in the Figma that the hover and active states should be different colors. I think what we have here is the correct _implementation_, as far as using `$theme-link-hover-color` and `$theme-link-active-color`, but it appears they're currently configured to the same color.

I think it might make sense to update that globally so that the link hover and active colors are distinct, if that's the intention. What do you think?

Reply:
Yes I would like to update `$theme-link-hover-color: 'primary-dark';` and `$theme-link-active-color: 'primary-darker';` globally, to match the unstyled button mockups in Figma.  Currently links have a hover color of `primary-darker` and no active color. @nickttng does that sound good to you too?

Reply:
@anniehirshman-gsa Yep, that sounds good